code/datums/components/footstep.dm

```diff
@@ -80,23 +81,29 @@
 	var/turf/simulated/floor/T = prepare_step()
 	if(!T)
 		return
+
+	// Single set sound file
 	if(isfile(footstep_sounds) || istext(footstep_sounds))
-		playsound(T, footstep_sounds, volume, falloff_distance = 1, vary = sound_vary)
+		playsound(T, footstep_sounds, volume, sound_vary)
```

Comment:
For all of these changes, here's the arguments for `playsound()`
https://github.com/ParadiseSS13/Paradise/blob/3f6eac48d17fcf336b1e2e21a12f56ee92b5abf5/code/game/sound.dm#L44
`vary` is the fourth argument so doesn't need to be named here, and `falloff_distance` is `1` by default.
https://github.com/ParadiseSS13/Paradise/blob/3f6eac48d17fcf336b1e2e21a12f56ee92b5abf5/code/__DEFINES/sound.dm#L27